pipeline_dp/__init__.py

```diff
@@ -6,3 +6,4 @@
 from pipeline_dp.pipeline_operations import LocalPipelineOperations
 from pipeline_dp.pipeline_operations import BeamOperations
 from pipeline_dp.pipeline_operations import SparkRDDOperations
+from pipeline_dp.accumulator import CompoundAccumulator
```

Comment:
Nit: remove from init.py

Explanation: CompoundAccumulator is not a part of Public API

Reply:
Sorry missed that.